services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/DefaultAttributeConverterProvider.java

```diff
@@ -159,7 +159,9 @@ public static Builder builder() {
         }
 
         if (type.tableSchema().isPresent()) {
-            converter = DocumentAttributeConverter.create(type.tableSchema().get(), type);
+            converter = DocumentAttributeConverter.<T>builder().tableSchema(type.tableSchema().get())
+                                                               .enhancedType(type)
+                                                               .build();
```

Comment:
Now we're not passing in an extra flag this has become change for the sake of change. My recommendation is we just leave it be.